scripts/airgap.sh

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/env bash
+set -eEuo pipefail
+
+# ==================================================================================================
+# TODO:
+#
+# Review (Victor): https://github.com/kubewarden/kwctl/pull/375#discussion_r1048740479
+# current approach doesn't use the GH release files kubewarden-crds-1.2.3_kubewarden-defaults-1.2.8_kubewarden-controller-1.2.8_policylist.txt
+# - it does
+# canonical locations of artifacts
+# - please explain (helm charts vs release?)
+#
+# TODO:
+# fix https://docs.kubewarden.io/operator-manual/airgap/requirements
+# blogpost - https://github.com/kubewarden/kubewarden.io/pull/153
+# fix documentation - https://github.com/kubewarden/docs/pull/150
+# - get image / policy list from pulled charts?
+# ==================================================================================================
+
+
+# Defaults
+IMAGES_TAR="kubewarden-images.tar"
+POLICIES_TAR="kubewarden-policies.tar.gz"
+
+PROG=$(basename "$0")
+USAGE="\
+Usage:  $PROG command [--parameters]
+
+  $PROG list [--cert-manager]
+  $PROG pull -l file.json [--dry]
+  $PROG push -l file.json -r registry[:port] [--insecure] [--dry]
+  $PROG install -l file.json -r registry[:port] [--insecure] [--dry]
+
+Parameters:
+  -l|--list       Json file with list of required dependencies
+  -r|--registry   Private registry where artifacts are pushed. Helm charts are configured for this registry
+  --dry           Do not perform actions, only print commands that would be executed
+  --cert-manager  Include cert-manager dependencies in the generated list
+  --insecure      For pull command skip kwctl certificate verification (an alternative to sources.yaml file)
+                  For install disable policyServer verification of registry
+
+Commands:
+  list     Generate json list of dependencies. You can redirect it to a file
+  pull     Pull components from list to current directory
+  push     Push components in list from current directory to local registry
+  install  Install helm charts configured with local registry
+
+Requirements: (https://docs.kubewarden.io/operator-manual/airgap/requirements)
+  - jq, kwctl, helm, docker
+  - kubewarden helm repository
+
+Examples:
+  Generate list of dependencies required by airgap. Save in file for other commands:
+    $PROG list --cert-manager | tee file.json
+
+  Pull images and policies and save them to archives in current directory:
+    $PROG pull --list file.json
+
+  Push images and policies from created archives to local registry. Run in DRY mode:
```

Comment:
```suggestion
  Push images and policies from created archives to local registry. Run in dry-run mode:
```

(DRY on is own, and in uppercase could be misunderstood as [Don't Repeat Yourself](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself))